examples/local_elasticsearch.conf

```diff
@@ -0,0 +1,26 @@
+input {
+  http_poller {
+    urls => {
```

Comment:
@andrewvc Yes, sure. My questioning was more of a : "what if I have a lot of complex request, how readable would that be?"

+1 for the ```:validate => lambda```